core/config/v2/docs/core.toml

```
@@ -176,6 +176,10 @@ SecureCookies = true # Default
 SessionTimeout = '15m' # Default
 # SessionReaperExpiration represents how long an API session lasts before expiring and requiring a new login.
 SessionReaperExpiration = '240h' # Default
+# HTTPMaxSize defines the maximum size for HTTP requests and responses made by the node server.
+HTTPMaxSize = '32768' # Default
```

Would including the implied bytes suffix be more clear?
```suggestion
HTTPMaxSize = '32768b' # Default
```